src/routes/home/components/DisciplineCard.tsx

```diff
@@ -63,9 +63,12 @@ const DisciplineCard = ({
   }
 
   if (!discipline) {
+    if (error?.message !== ForbiddenError && error?.message !== NetworkError) {
+      return null
+    }
     return (
       <Card>
-        {error?.message === ForbiddenError && isTypeLoadProtected(identifier) ? (
+        {error.message === ForbiddenError && isTypeLoadProtected(identifier) ? (
```

Comment:
I think it would be better if we show this part (with the button to delete the discipline) for all errors except Network error. Otherwise users may be confused, why a discipline suddenly has disappeard.